asm/overlay_10_rodata_022C464C.s

```diff
@@ -428,20 +428,20 @@ ov10_022C487C:
 	.global ov10_022C4880
 ov10_022C4880:
 	.byte 0x7F, 0x00, 0x7F, 0x00
-	.global ov10_022C4884
-ov10_022C4884:
+	.global gCringeTurnRange
```

Comment:
Globals here are usually named with SCREAMING_SNAKE_CASE, like `SPEED_BOOST_TURN_RANGE` below.

Reply:
I'd rather the globals be consistent across the repos too